apps/relayer/src/chain/transaction.service.ts

```diff
@@ -228,4 +242,21 @@ export class TransactionService implements OnModuleInit, OnModuleDestroy {
       }
     }
   }
+
+  private async updateGasPrice() {
+    try {
+      const gasPriceMinimum = await this.kit.contracts.getGasPriceMinimum()
+      const rawGasPrice = await gasPriceMinimum.getGasPriceMinimum(ZERO_ADDRESS)
```

Comment:
Should we perform a sanity check on this response to make sure it's not above some threshold? If the user is on data-saver mode and trusting a full node, a malicious full-node could return unreasonable gas prices to steal the user's funds

Reply:
This isn't a data-saver mode scenario but nonetheless I think it's worthwhile to have a cap :+1: